Didi/update datagrid docs (#923)
telerik/maui-docs: knowledge-base/gauge-rangesdefinition-gradientstop-bindable-properties.md

---

> title: Bindings for properties inside the RadGauge GaugeRangesDefinition and GradientStop classes

Reviewer: Use title case for the titles and headings.

description: Learn how to apply bindings for the properties inside the GaugeRangesDefinition and GradientStop classes of the Gauge for .NET MAUI.
type: how-to
page_title: How to make the properties inside the GaugeRangesDefinition and GradientStop Bindable.
slug: gauge-rangesdefinition-gradientstop-bindable-properties
tags: gauge, ranges, bindable properties, .net maui
res_type: kb
---

## Environment

| Product | Version | Author |
| --- | --- | --- | 
| Gauge for .NET MAUI | 7.1.0 | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova) | 

## Description

> When using a horizontal or vertical Gauge in a .NET MAUI application, you may want to bind the values for the offset, color properties inside the `RadGradientStop` or bind the values for the `StartThickness` and `EndThickness` for the `GaugeRangesDefinition`. This article demonstrates how to set bindings for the properties inside the `GaugeRangesDefinition` and `RadGradientStop` classes.

Reviewer: ```suggestion
When using a horizontal or vertical Gauge in a .NET MAUI application, you may want to bind the values for the offset, the color properties inside the `RadGradientStop`, or the values for the `StartThickness` and `EndThickness` for the `GaugeRangesDefinition`. This article demonstrates how to set bindings for the properties inside the `GaugeRangesDefinition` and `RadGradientStop` classes.
```

This KB article also answers the following questions:
- How can I set bindings for the `Offset` and `Color` properties inside the `RadGradientStop`?
- How can I set bindings for the `StartThickness` and `EndThickness` properties inside the `GaugeRangesDefinition`?

## Solution

> The properties inside the `GaugeRangesDefinition` and `RadGradientStop` classes are not bindable. If you want to apply binding you will need an attached property.

Reviewer: ```suggestion
The properties inside the `GaugeRangesDefinition` and `RadGradientStop` classes are not bindable. To apply binding, use an attached property.
```
